app/workers/update_for_you_worker.rb

```diff
@@ -81,7 +81,7 @@ def following_status
   # Indirect Follows
   def indirect_following_status
     user_setting = @user[:for_you_settings]
-    return if user_setting[:friends_of_friends].zero?
+    return if user_setting[:friends_of_friends].zero? || user_setting[:type] == 'public'
 
```

Comment:
additional guards from running beta following/indirect_following if they are 'on' `1`, the user must also be of type 'personal'